src/components/Sidebar/Contacts/Contacts.js

```diff
@@ -13,7 +13,9 @@ type Props = {
 const Contacts = ({ contacts }: Props) => (
   <div className={styles['contacts']}>
     <ul className={styles['contacts__list']}>
-      {Object.keys(contacts).map((name) => (
+      {Object.keys(contacts).map((name) => 
+      contacts[name] === '' ? ('') : 
```

Comment:
I think we should just check if `contacts[name]` is truthy here - that way, we can catch it if the value is `undefined` or `null`, too.

Also, use `null` instead of an empty string for the empty case. I think React still creates a `<span>` for an empty string, which we don't want.

Reply:
Thank you for the feedback :) I fixed this but Prettier changed way more than expected. Sadly, I did not notice before pushing. To avoid messing up the git history I can create a new pull request containing only one commit if wanted?

Reply:
Sure, if you make a new PR just close this one and refer to it from the new one!

Reply:
Code and also the git history in my fork is now fixed. Again thank you for the comments :)